File: official/docs/java/carrierTypes/RetrieveCarrierTypes.java
import com.easypost.EasyPost;
import java.util.List;

public class RetrieveCarrierTypes {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        List<CarrierType> carrierTypes = CarrierType.all();

        System.out.println(carrierTypes);
    }
}

File: official/docs/csharp/carrier_types/retrieve-carrier-types.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            EasyPost.ClientManager.SetCurrent(apiKey);

            List<CarrierType> carrierTypes = await CarrierType.All();

            Console.WriteLine(JsonConvert.SerializeObject(carrierTypes, Formatting.Indented));
        }
    }
}
